launcher/src/functionalTest/kotlin/com/cognifide/gradle/aem/launcher/LauncherTest.kt

```diff
@@ -9,7 +10,13 @@ import java.util.concurrent.TimeUnit
 class LauncherTest {
 
     @Test
-    fun shouldRunProperly() = makeTestProject("tasks", null) { launch("tasks") }
+    fun shouldRunProperly() = makeTestProject("tasks", null) {
+        launch("tasks")
+        assertEquals(
+            "https://services.gradle.org/distributions/gradle-7.5.1-bin.zip",
```

Comment:
I presume this will stay constant until we update the gradle version used for building but I’m not 100% positive… If so, are you OK with updating this from time to time or should we make the assertion dynamic?

Reply:
I quickly tested it and indeed, it updates when we update the gradle wrapper used for building.

Reply:
generally I prefer avoiding hardcoding such stuff at all. maintaining GAP is already not so easy process and there are many places to check after each upgrade

Reply:
OK, I switched to reading the `gradle-wrapper.properties` of the main project to compare it with the generated project. That should be stable on updates.